src/Compilers/Core/Portable/Serialization/FixedObjectBinder.cs

```diff
@@ -1,48 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Immutable;
-
-namespace Roslyn.Utilities
-{
-    /// <summary>
-    /// An <see cref="ObjectBinder"/> with a fixed set of type and reader mappings.
-    /// </summary>
-    internal class FixedObjectBinder : ObjectBinder
```

Comment:
Removed all subclassing of ObjectBinder.  It had one normal subclass and another purely to represent an 'empty, immutable' binder.  This was replaced with the real binder and 'null' used to the represent the latter.